caf-logging-tomcat-juli/LICENSE

```diff
@@ -0,0 +1,88 @@
+Eclipse Public License (EPL)
+----------------------------
+
+THE ACCOMPANYING PROGRAM IS PROVIDED UNDER THE TERMS OF THIS ECLIPSE PUBLIC LICENSE ("AGREEMENT"). ANY USE, REPRODUCTION OR DISTRIBUTION OF THE PROGRAM CONSTITUTES RECIPIENT'S ACCEPTANCE OF THIS AGREEMENT.
```

Comment:
I don't understand why this license file is here.  Are we introducing something that isn't our code?

Reply:
Please suggest to me how to handle these license files when modifying third-party APIs?
I don't know how to include license files when creating new module with their source.